spark-doris-connector/src/main/java/org/apache/doris/spark/cfg/ConfigurationOptions.java

```diff
@@ -90,6 +90,16 @@ public interface ConfigurationOptions {
 
     int DORIS_SINK_BATCH_INTERVAL_MS_DEFAULT = 50;
 
+    String DORIS_SINK_MAX_BLOCKING_TIMES = "doris.sink.max.blocking.times";
```

Comment:
Is this parameter similar to sink.max-retries?

Reply:
'sink.max-retries' is to specify the maximum number of retries for other exceptions, and 'doris.sink.max.blocking.times' is to specify the maximum number of retries when 'doris.sink.block.trigger.keys' appears, Often in order to ensure the success of data import, 'doris.sink.max.blocking.times' is usually set larger